src/docfx/serve/lsp/credentialRefresh/CredentialRefreshParams.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Threading.Tasks;
+using MediatR;
+using OmniSharp.Extensions.JsonRpc;
+
+namespace Microsoft.Docs.Build
+{
+    [Parallel]
+    [Method("docfx/userCredentialRefresh", Direction.ServerToClient)]
+    public record CredentialRefreshParams : IRequest<CredentialRefreshResponse>
+    {
+        public CredentialType Type { get; init; }
+    }
+
+    public record CredentialRefreshResponse
+    {
+        public CredentialRefreshResult? Result { get; init; }
+
+        public ResponseError? Error { get; init; }
```

Comment:
Is this the standard way of handling errors in LSP (or jsonrpc)? There is a dedicated [Error Object](https://www.jsonrpc.org/specification#error_object) in jsonrpc to represent error and it seems to be at a different level.

```suggestion
       ~~public ResponseError? Error { get; init; }~~
```